java/jdbc/SessionlessTransactions/src/main/java/com/oracle/jdbc/samples/sessionlesstxns/dto/RemoveTicketRequest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
```

Comment:
Use the same license presentation as in:
https://github.com/oracle/oci-react-samples/blob/spring-appstack/src/main/java/com/oracle/todoapp/mytodolist/ToDoItem.java